### ISSUE TITLE
Distribute of course materials with `usethis::use_course()`

### ISSUE THREAD

**Issue by maurolepore** [datacarpentry/R-ecology-lesson#384]
The __usethis__ package recently introduced (in v.1-3-0) the function `usethis::use_course()`:

![](https://i.imgur.com/gmuJyiA.png)

-- From https://www.tidyverse.org/articles/2018/02/usethis-1-3-0/

**Comment by fmichonneau** [comment]
Yes, I've considered it. Unfortunately, at this time it's not possible to use it to distribute the datasets as the figshare server returns the wrong mime type object on the URL pointing to the zip files which makes `use_course()` choke. We might revisit this later though once I have a chance to investigate this in more detail.
